source/MaterialXTest/Runtime.cpp

```diff
@@ -498,21 +520,44 @@ TEST_CASE("Runtime: CoreIo", "[runtime]")
         texcoord1_out.connectTo(tiledimage1_texcoord);
         texcoord1_index.getValue().asInt() = 2;
 
-        mx::RtFileIo(stage.getObject()).write(stage.getName().str() + "_export.mtlx");
+        // Test write and read-back with explicit xincludes
```

Comment:
Since write adds in xincludes the definitions are read in as required if not there. If they are already there then we need to make sure to "skip duplicates" (e.g. in stream test)